purview/models/koji.py

```diff
@@ -0,0 +1,55 @@
+from neomodel import (
+    StructuredNode, StringProperty, IntegerProperty, UniqueIdProperty,
+    BooleanProperty, DateTimeProperty, FloatProperty, RelationshipTo)
+
+
+class KojiBuild(StructuredNode):
+    id_ = UniqueIdProperty(db_property='id')
+    tasks = RelationshipTo('KojiTask', 'TRIGGERS')
+    creation_event_id = IntegerProperty(required=True)
+    volume_id = IntegerProperty()
+    owner_id = IntegerProperty()
+    package_id = IntegerProperty(required=True)
+    task_id = IntegerProperty(required=True)
+    build_id = IntegerProperty()
+    name = StringProperty()
+    owner_name = StringProperty()
+    package_name = StringProperty(required=True)
+    version = StringProperty(required=True)
+    release = StringProperty(required=True)
+    epoch = StringProperty()
+    nvr = StringProperty()
+    source = StringProperty()
+    state = IntegerProperty(required=True)
+    volume_name = StringProperty()
+    start_time = DateTimeProperty()
+    creation_time = DateTimeProperty(required=True)
+    completion_time = DateTimeProperty()
+    start_ts = FloatProperty()
+    creation_ts = FloatProperty(required=True)
+    completion_ts = FloatProperty()
+    extra = StringProperty()
```

Comment:
Let's parse this in the scraper and a `scratch` property on the KojiBuild node.

Reply:
`scratch` should be a property of `KojiTask`. A `KojiBuild` will never be created for a scratch task. This `extra` property is actually extra metadata in json format, usually provided by a `Content Generator` when it imports a build from another system. I would suggest retaining it, it might be useful for handling container and Middleware builds.